packages/account/src/Sections/Security/DeactivateAccount/deactivate-account-reason.jsx

```diff
@@ -140,7 +140,7 @@ class DeactivateAccountReason extends React.Component {
                 this.setState({ remaining_characters: 0 });
             }
 
-            if (!/^[0-9A-z .,'-]*$/.test(final_value)) {
+            if (!/^[0-9A-z .,'-\s]*$/.test(final_value)) {
```

Comment:
This looks like irrelevant to the PR's purpose, would you please clarify?

Reply:
You are right, looks like it's stays from previous task. I will be more accurate with branches. Changed that.